### PR TITLE
Default async to true, headers undefined is ok

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   },
   "devDependencies": {
     "web-component-tester": "~2.2.3",
-    "simple-fixture": "polymer/test-fixture"
+    "test-fixture": "polymer/test-fixture",
```
Comment (justinfagnani) [inline bower.json]: style: should we alphabetize to minimize diffs later?

File: core-request.html [hunk 2]
```diff
@@ -141,17 +141,19 @@
       }.bind(this));
 
       xhr.open(
-        options.method,
+        options.method || 'GET',
         options.url,
-        options.async
+        options.async !== false
```
Comment (justinfagnani) [inline core-request.html]: :crying_cat_face: how about just hardcode this to `false`. Sync XHR is bad and you should feel bad for enabling it. :P

File: core-request.html [hunk 3]
```diff
@@ -141,17 +141,19 @@
...
       );
 
-      Object.keys(options.headers).forEach(function (requestHeader) {
-        xhr.setRequestHeader(
-          requestHeader,
-          options.headers[requestHeader]
-        );
-      }, this);
+      if (options.headers) {
```
Comment (justinfagnani) [inline core-request.html]: :clap:

**Comment by justinfagnani** [comment]
LGTM

File: bower.json (hunk 1 above)
Comment (cdata) [inline bower.json]: Will fix.

File: core-request.html (hunk 2 above)
Comment (cdata) [inline core-request.html]: File an issue, we can discuss further :train:

**Pull request merged**